lib/ansible/modules/remote_management/ucs/ucs_timezone.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: ucs_timezone
+
+short_description: Configures timezone on Cisco UCS Manager
+
+version_added: "2.6"
+
+description:
+    - Configures time zone on Cisco UCS Manager.
+    - Examples can be used with the UCS Platform Emulator U(https://communities.cisco.com/ucspe).
+
+options:
+    state:
+        description:
+            - If C(present), will verify timezone is set (persent) and will set (create) if needed.
+            - If C(absent), will verify timezone is unset (absent) and will unset (delete) if needed.
+        choices: [present, absent]
+        default: absent
+        required: true
+
+    admin_state:
+        description: admin state indicates if the timezone confguration is enabled (utilized by UCS Manager)
+            or disabled (not utilized by UCS Manager)
+        choices: ['disabled', 'enabled']
+        default: "enabled"
+        required: false
+
+    timezone:
+        description:
+            - The timezone name. Time zone names are from the tz database U(https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
+            - The timezone name is case sensitive.
+            - This name can be between 0 and 510 alphanumeric characters.
+            - You cannot use spaces or any special characters other than
+            - "\"-\" (hyphen), \"_\" (underscore), \"/\" (backslash)."
+        required: false
+
+    descr:
```

Comment:
The latest modules have spelled out 'description' with an alias to 'descr'.  You'll then want to use 'description' in all the code below.  vnic_template should be a good reference module to look at for how the alias is handled: https://github.com/ansible/ansible/pull/35136/files

Reply:
I have changed descr to description based on supplied reference pr